sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -526,6 +526,15 @@ object SQLConf {
     .checkValue(_ >= 1, "The shuffle hash join factor cannot be negative.")
     .createWithDefault(3)
 
+  val INITIAL_NUM_PARTITIONS = buildConf("spark.sql.limit.initialNumPartitions")
```

Comment:
Rename to `LIMIT_INITIAL_NUM_PARTITIONS` for consistency with `LIMIT_SCALE_UP_FACTOR`